rbd-target-api.py

```diff
@@ -2500,15 +2504,21 @@ def target_ready(gateway_list):
             api.get()
```

Comment:
When we initialize target_state at the beginning of this function, do we need to also init status_iscsi and status_api to both 'UP'?

Reply:
Fixed, thanks.